### ISSUE TITLE
[REQUEST] Grade

### ISSUE THREAD

**Issue by Exu9x** [qbcore-framework/qb-policejob#500]
Can you please set back that police can take out weapons and cars by grades? Because it's not common when recruit can take rifle

**Comment by jay6598** [comment]
yes i like the job grades like it was, i don't like that higher grades can see lower items and cars if you have a long list it not easy to work
